lib/gyoku/prettifier.rb

```diff
@@ -4,8 +4,9 @@ module Gyoku
   class Prettifier
     DEFAULT_INDENT = 2
     DEFAULT_COMPACT = true
+    DEFAULT_ATTRIBUTE_QUOTE = '"'
 
-    attr_accessor :indent, :compact
+    attr_accessor :indent, :compact, :attribute_quote
```

Comment:
This accessor is not used in this PR.

Is it meant to be used outside this class?